@alikins after the commit 8fb5ce5, now it fails with the following error:

`$ ./build/scripts-2.7/ansible r7node1 -m pacemaker_cluster -a 'state=cleanup' -b
r7node1 | FAILED! => {
    "changed": false, 
    "failed": true, 
    "module_stderr": "Shared connection to r7node1 closed.\r\n", 
    "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_Oi8EDR/ansible_module_pacemaker_cluster.py\", line 228, in <module>\r\n    main()\r\n  File \"/tmp/ansible_Oi8EDR/ansible_module_pacemaker_cluster.py\", line 224, in main\r\n    out=cluster_state)\r\nUnboundLocalError: local variable 'cluster_state' referenced before assignment\r\n", 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
`

If you add the following line 223 it will correctly executed(I tested it over a RHEL 7.2 pacemaker cluster)
```
    221     if state in ['cleanup']:
    222         clean_cluster(module, timeout)
    223         cluster_state = get_cluster_status(module)
    224         module.exit_json(changed=True,
    225                  out=cluster_state)
```

--- a/lib/ansible/modules/clustering/pacemaker_cluster.py
+++ b/lib/ansible/modules/clustering/pacemaker_cluster.py
@@ -220,8 +220,9 @@ def main():
 
     if state in ['cleanup']:
         clean_cluster(module, timeout)
+        cluster_state = get_cluster_status(module)
         module.exit_json(changed=True,
-                 out=cluster_state)
+                         out=cluster_state)
 
 
 if __name__ == '__main__':

